internal/config/config.go

```diff
@@ -37,7 +37,14 @@ type hclConfig struct {
 
 // Runner is the configuration for supporting runners in this project.
 type Runner struct {
-	// Enabled is whether or not runners are enabled. If this is false
+	// Profile is the on-demand runner configuration.
+	Profile string `hcl:"profile"`
```

Comment:
Small nit, but should this be `ProfileId` or `ProfileName`? I expected `Profile` to be a full proto message of an ODR profile when I was reading this else where (as the comment here suggests), but it's the name of the profile.

We could even keep it where the HCL var is `profile` but internally we call it `ProfileName`.

Reply:
Hmm I'd prefer keeping it as "profile" for consistency, but don't feel strongly

Reply:
Sounds good, yeah probably good to keep the HCL variable the same as how we refer internally! And I think the docs update will help with the confusion :+1: